tools/slang-unit-test/unit-test-bindless-space-reflection.cpp

```
@@ -75,10 +81,184 @@ SLANG_UNIT_TEST(bindlessSpaceReflection)
+    _checkBindlessSpaceReflection(userSource, _expectReservedBindlessSpaceAt(2), true);
+}
+
+SLANG_UNIT_TEST(bindlessSpaceMetadataHLSLResourceAndSamplerHandles)
+{
+    const char* userSource = R"(
+        RWStructuredBuffer<float> gOutput;
+
+        struct P
+        {
+            Texture2D.Handle t;
+            SamplerState.Handle s;
```

Cppcheck (2.20.0)

[style] The function 'addHash' is never used.

(unusedFunction)